src/collective/fingerpointing/logger.py

```diff
@@ -12,33 +14,53 @@
 FORMAT = '%(asctime)s - %(levelname)s - %(message)s'
 
 # by default, the audit log will use the same location used for the event log
-eventlog = getattr(getConfiguration(), 'eventlog', None)
+zopeConf = getConfiguration()
```

Comment:
@pcdummy can't we get from here somehow the location of the `var/log` directory?